RUM-6885 Add the docs for RumMonitor#addViewLoadingTime API (#25914)
DataDog/documentation: content/en/real_user_monitoring/mobile_and_tv_monitoring/advanced_configuration/ios.md

For more details and available options, filter the [relevant file on GitHub][9] for the `DDRUMMonitor` class.

### Notify the SDK that your view finished loading

We are tracking the time it takes for your view to load. To notify the SDK that your view has finished loading you will need to call the `addViewLoadingTime` method
through the `RUMMonitor` instance. This method should be called when your view is fully loaded and ready to be displayed to the user:

{{< tabs >}}
{{% tab "Swift" %}}
```swift
func onHeroImageLoaded() {
    let rum = RUMMonitor.shared()
    rum.addViewLoadingTime(override: false | true)
}
```
{{% /tab %}}
{{% tab "Objective-C" %}}
```objective-c
- (void)onHeroImageLoad {
    [[DDRUMMonitor shared] addViewLoadingTimeWithOverride:NO | YES];
}
```
{{% /tab %}}
{{< /tabs >}}

> 

Reviewer: This would be a good addition for Flutter as well.

For more details and available options, filter the [relevant file on GitHub][9] for the `DDRUMMonitor` class.

### Notify the SDK that your view finished loading

> We are tracking the time it takes for your view to load. To notify the SDK that your view has finished loading you will need to call the `addViewLoadingTime(override:)` method
> through the `RUMMonitor` instance. This method should be called when your view is fully loaded and displayed to the user:

Reviewer 2: ```suggestion
iOS RUM tracks the time it takes for your view to load. To notify the SDK that your view has finished loading, call the `addViewLoadingTime(override:)` method
through the `RUMMonitor` instance. Call this method when your view is fully loaded and displayed to the user:
```

{{< tabs >}}
{{% tab "Swift" %}}
```swift
@_spi(Experimental)
import DatadogRUM

func onHeroImageLoaded() {
    let rum = RUMMonitor.shared()
    rum.addViewLoadingTime(override: false)
}
```
{{% /tab %}}
{{% tab "Objective-C" %}}
```objective-c
- (void)onHeroImageLoad {
    [[DDRUMMonitor shared] addViewLoadingTimeWithOverride:NO | YES];
}
```
{{% /tab %}}
{{< /tabs >}}

> Use `override` option to replace previously calculated loading time for current view.

Reviewer 2: ```suggestion
Use the `override` option to replace the previously calculated loading time for the current view..
```

> Once the view loading time is sent, the loading time is accessible as `@view.loading_time` and will be visible also in the RUM UI.
> Please note that this API is still experimental and might change in the future.

Reviewer 2: ```suggestion
After the loading time is sent, it is accessible as `@view.loading_time` and is visible in the RUM UI.
Please note that this API is still experimental and might change in the future.
```
